doc/sphinx/addendum/extraction.rst

```diff
@@ -180,11 +180,14 @@ and commands:
 
 .. flag:: Extraction AutoInline
 
-   Default is on. The extraction mechanism inlines the :term:`bodies <body>` of
+   Default is off. When enabled, the extraction mechanism inlines the :term:`bodies <body>` of
    some defined :term:`constants <constant>`, according to some heuristics
    like size of bodies, uselessness of some arguments, etc.
-   Those heuristics are not always perfect; if you want to disable
-   this feature, turn this :term:`flag` off.
+
+   Even when this flag is off, recursors, projections, and a few
+   specific constants such as `andb` and `orb` (for the lazy
+   behaviour) and well founded recursion combinators are still
```

Comment:
```suggestion
   behavior) and well founded recursion combinators are still
```

The American spelling.

Reply:
I'm not american

Reply:
I had no idea.  The documentation uses American spelling.  This was not my decision.

Reply:
It's your decision to nitpick over it.